tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -47,7 +47,6 @@ Feature: Sharing files and folders with internal users
       | set-role    | expected-role | permissions-folder        | permissions-file |
       | Viewer      | Viewer        | read                      | read             |
       | Editor      | Editor        | read,change,create,delete | read,change      |
-      | Custom Role | Viewer        | read                      | read             |
```

Comment:
This way we are losing the coverage also for folders, aren't we? Can you pls add a new step where you explicitly share a folder with that role? Or do you have better idea @individual-it ?